src/components/Training/Modal.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import is from 'styled-is'
+import ReactMarkdown from 'react-markdown'
+import { Padding } from 'styled-components-spacing'
+import StyledLink from '../styledLink'
+import Image from '../Common/Image'
+import remcalc from 'remcalc'
+import { H2, Paragraph } from '../Typography'
+import { Grid, Row, Col } from '../grid'
+
+const Wrapper = styled.div`
+  width: 100vw;
+  height: 100vh;
+  background-color: #ffffff;
+  position: fixed;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  z-index: 9999;
+  transition: transform 0.3s, opacity 0.3s;
+  transform: scale(0.7);
+  visibility: hidden;
+
+  ${is('visible')`
+    transform: scale(1);
+    opacity: 1;
+    visibility: visible;
+    overflow: scroll;
+  `};
+`
+
+const Close = styled.button`
```

Comment:
The differences are not really minor. I think it makes sense to separate them because the header one is not showing on mobile and it's reverse and it would just be a bunch pf props